contracts/protocol/facets/ProtocolInitializationHandlerFacet.sol

```diff
@@ -133,6 +136,20 @@ contract ProtocolInitializationHandlerFacet is IBosonProtocolInitializationHandl
         require(protocolStatus().version == bytes32("2.2.0"), WRONG_CURRENT_VERSION);
     }
 
+    /**
+     * @notice Initializes the version 2.3.0.
+     *
+     * Reverts if:
+     *  - Current version is not 2.2.1
+     *  - There are already twins. This version adds a new mapping for twins which make it incompatible with previous versions.
+     */
+    function initV2_3_0() internal view {
+        // Current version must be 2.2.1
+        require(protocolStatus().version == bytes32("2.2.1"), WRONG_CURRENT_VERSION);
+
+        require(protocolCounters().nextTwinId == 1, "Should not have any twins yet");
```

Comment:
Can you please move the revert reason to `BosonConstants.sol`?

Reply:
done